packages/react-devtools-shared/src/devtools/views/Profiler/CommitFlamegraph.js

```diff
@@ -140,7 +159,7 @@ function CommitFlamegraph({chartData, commitTree, height, width}: Props) {
     }),
     [
       chartData,
-      hoverFiber,
+      setHoveredFiberData,
```

Comment:
This looks wrong. The dependencies array has `setHoveredFiberData ` but the memoized value references `hoveredFiberData`.

I wonder why our lint rule isn't complaining about this.